stanford-earth/SE3_BLT
# EARTH-299: Enable double filmstrip and film card by default.

# READY FOR REVIEW

# Summary
- Enable Double filmstrip paragraph type

# Needed By (Date)
- End of Sprint

# Steps to Test

1. Checkout this branch & same branch in [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/65) & `EARTH-296-callout-filmstrip` branch of [stanford_components](https://github.com/SU-SWS/stanford_components/pull/69) (If it hasn't already been merged)
2. config import & cache clear
3. create complex page with "Double Filmstrip paragraph"
4. Verify it looks normal.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/69
- https://github.com/SU-SWS/stanford_paragraph_types/pull/65

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
